protocols/gossipsub/src/behaviour/tests.rs

```diff
@@ -108,28 +107,58 @@ where
 
         (gs, peers, topic_hashes)
     }
-}
 
-impl<D, F> InjectNodesBuilder<D, F>
-where
-    D: DataTransform + Default + Clone + Send + 'static,
-    F: TopicSubscriptionFilter + Clone + Default + Send + 'static,
-{
-    pub fn create_network(&self) -> (Gossipsub<D, F>, Vec<PeerId>, Vec<TopicHash>) {
-        self.build().unwrap().create_network()
+    fn peer_no(mut self, peer_no: usize) -> Self {
+        self.peer_no = peer_no;
+        self
+    }
+
+    fn topics(mut self, topics: Vec<String>) -> Self {
+        self.topics = topics;
+        self
+    }
+
+    fn set_to_subscribe(mut self, to_subscribe: bool) -> Self {
```

Comment:
I am guessing that clippy made you rename this because it doesn't follow some convention?

Sorry to go back on this but I'd prefer all builder functions to be consistent and just slap a `#[allow(clippy::lint_name)]` on here. Otherwise we have this unnecessary diff when all we wanted was to replace the macro :)